src/app/authentication/authentication.component.scss

```diff
@@ -1,104 +1,102 @@
 @import url(https://fonts.googleapis.com/css?family=Roboto:300);
+@import url("//maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css");
```

Comment:
those two @import statements are public for the whole app. 
they need to be in a general style file.